packages/ldap-integration/.env.example

```diff
@@ -0,0 +1,21 @@
+## Uncomment the lines as needed
+
+NAME=local
+NODE_ENV=local
+PRODUCTION=false
+
+## Server URLs
+API=https://localhost:8080
+GRAPHQL_API=https://localhost:8080/playground
+
+## Database
+DB_PATH=localhost
+DB_NAME=db_name
+# DB_USER=
+# DB_PASSWORD=
+
+## Cookie Parser
+COOKIE_PARSER=LDAP_SECRET
+
+## Authorization Token
+# AUTH_TOKEN=
```

Comment:
Shouldn't we be updating this file as per the needs of our Micro-service?

Reply:
We are not going to commit the actual `.env` file  that will be managed seperatly

Reply:
All the host related information belongs there